Formula/bash-preexec.rb

```diff
@@ -1,5 +1,5 @@
 class BashPreexec < Formula
-  desc "preexec and precmd functions for Bash just like Zsh."
```

Comment:
It's also just occurred to me that we should probably enforce the lack of full-stops at the end of descriptions. What do you think?